### PR TITLE
[Keymap] Add new keymap for ut472

### PR DESCRIPTION
**Pull request by stefanopace**
## Description
I've added a new keymap for the ut472 pcb

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/ut472/keymaps/stefanopace/config.h [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#pragma once
```
Comment (drashna) [inline keyboards/ut472/keymaps/stefanopace/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

File: keyboards/ut472/keymaps/stefanopace/config.h [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/ut472/keymaps/stefanopace/config.h:3]: This shouldn't be needed. 
```suggestion
```

File: keyboards/ut472/keymaps/stefanopace/keymap.c [hunk 3]
```diff
@@ -0,0 +1,169 @@
+#include QMK_KEYBOARD_H
```
Comment (drashna) [inline keyboards/ut472/keymaps/stefanopace/keymap.c:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

**Comment by stefanopace** [comment]
Thanks,
I've added the license headers and removed the unused include. I confirm that the keymap compile and works without the include.

File: keyboards/ut472/keymaps/stefanopace/config.h (hunk 1 above)
Comment (stefanopace) [reply keyboards/ut472/keymaps/stefanopace/config.h]: added

File: keyboards/ut472/keymaps/stefanopace/config.h (hunk 2 above)
Comment (stefanopace) [reply keyboards/ut472/keymaps/stefanopace/config.h]: removed, the keymap still compile and works

File: keyboards/ut472/keymaps/stefanopace/keymap.c (hunk 3 above)
Comment (stefanopace) [reply keyboards/ut472/keymaps/stefanopace/keymap.c:17]: added

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Comment by drashna** [comment]
Thanks!

**Pull request merged**